_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -357,4 +150,8 @@ class HealthCareApp extends React.Component {
   }
 }
 
+// TODO(awong): Hack to allow access to the store for now while migrating.
+// All uses of this.context.store in this file are WRONG!!!
+HealthCareApp.contextTypes = { store: React.PropTypes.object };
```

Comment:
I think I just need to wrap HealthCareApp in a `connect` higher order component like I did with the sections. This change just got to big for me to tinker with it yet though.